app/js/systemoverview.js

```diff
@@ -0,0 +1,128 @@
+    ConcurrentSyncReportRuns: { Title: "Concurrent Sync Report Runs", Description: "Concurrent synchronous report runs via REST API" },
+    DailyAsyncApexExecutions: { Title: "Daily Async Apex Executions", Description: "Daily asynchronous Apex method executions (batch Apex, future methods, queueable Apex, and scheduled Apex)" },
+    DailyBulkApiRequests: { Title: "Daily Bulk API Requests" },
+    DailyDurableGenericStreamingApiEvents: { Title: "Daily Durable Generic Streaming Api Events" },
+    DailyDurableStreamingApiEvents: { Title: "Daily Durable Streaming Api Events" },
+    DailyGenericStreamingApiEvents: { Title: "Daily Generic Streaming Api Events", Description: "Daily generic streaming events (if generic streaming is enabled for your org)" },
+    DailyStreamingApiEvents: { Title: "Daily Streaming Api Events" },
+    DailyWorkflowEmails: { Title: "Daily Workflow Emails" },
+    DurableStreamingApiConcurrentClients: { Title: "Durable Streaming Api Concurrent Clients" },
+    HourlyAsyncReportRuns: { Title: "Hourly Async Report Runs", Description: "Hourly asynchronous report runs via REST API" },
+    HourlyDashboardRefreshes: { Title: "Hourly Dashboard Refreshes", Description: "Hourly dashboard refreshes via REST API" },
+    HourlyDashboardResults: { Title: "Hourly Dashboard Results", Description: "Hourly REST API requests for dashboard results" },
+    HourlyDashboardStatuses: { Title: "Hourly Dashboard Statuses", Description: "Hourly dashboard status requests via REST API" },
+    HourlyODataCallout: { Title: "Hourly OData Callout" },
+    HourlySyncReportRuns: { Title: "Hourly Sync Report Runs", Description: "Hourly synchronous report runs via REST API" },
+    HourlyTimeBasedWorkflow: { Title: "Hourly Time Based Workflow" },
+    MassEmail: { Title: "Mass Email", Description: "Daily number of mass emails that are sent to external email addresses by using Apex or Force.com APIs" },
+    SingleEmail: { Title: "Single Email", Description: "Daily number of single emails that are sent to external email addresses by using Apex or Force.com APIs" },
+    StreamingApiConcurrentClients: { Title: "Streaming Api Concurrent Clients" }
+};
+
+BfsSystemOverview.init = function () {
+    var sessionId = BfsSystemOverview.getSessionId();
+    BfsSystemOverview.getOrgLimits(sessionId);
+}
+
+BfsSystemOverview.getSessionId = function () {
+    return document.cookie.match("sid=([^;]*)")[1];
+}
+
+BfsSystemOverview.getOrgLimits = function (sessionId) {
+    $.ajax({
+        url: '/services/data/v40.0/limits/',
+        headers: { 'Authorization': 'Bearer ' + sessionId }
+    })
+    .always(BfsSystemOverview.processOrgLimitsResponse);
+}
+
+BfsSystemOverview.processOrgLimitsResponse = function (data, textStatus, jqXHR) {
+    if (textStatus == "success") {
+        BfsSystemOverview.showOrgLimitsUI(data);
+    } else {
+        console.log(jqXHR)
```

Comment:
Just thought it may be worthwhile for someone to see why it AJAX request failed even though it only shows in the dev console. Especially if someone submits a Boostr bug issue.